http-client-tck/src/main/java/io/micronaut/http/client/tck/tests/HttpMethodPostTest.java

```diff
@@ -60,7 +61,7 @@ String response() {
         }
 
         @Post("/object-body")
-        String person(Person person) {
+        String person(@Body Person person) {
             return person.getName() + ":" + person.getAge();
```

Comment:
Are you sure we want this change? We might break some code without it. It might be a good idea not to allow "blind" parameters.

Reply:
I feel like this might be a breaking change

Reply:
it is definitely a breaking change, but graeme and i decided it was worth doing to avoid ambiguity in binding